content/docs/command-reference/remote/modify.md

```diff
@@ -292,38 +336,22 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   $ dvc remote modify myremote projectname myproject
   ```
 
-- `credentailpath` - path of the file that contains
-  [service account](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account)
-  key. See **Accessing data with a service account** section below.
+- `credentailpath` - path to the file that contains the
+  [service account key](/doc/user-guide/setup-google-drive-remote#access-google-drive-remote-with-a-service-account).
+  Make sure that the service account has read/write access (as needed) to the
+  file structure in the remote `url`.
 
   ```dvc
-  $ dvc remote modify myremote credentailpath "/home/.../project-XXXXXXX.json"
+  $ dvc remote modify \
+        myremote credentailpath "/home/.../project-XXXXXXX.json"
   ```
 
-**Accessing data with a service account:**
```

Comment:
why removing this?

Reply:
Just refactored so all the info. needed is in the param descriptions.

Reply:
Discussing in https://github.com/iterative/dvc.org/pull/1096#discussion_r402006323. Resolving here.